apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,9 +618,9 @@ var ConversationView = {
         this.draft = Threads.get(threadId).getDraft();
 
         if (this.draft) {
-          this.draft.isEdited = false;
-
           Compose.fromDraft(this.draft);
+
+          this.draft.isEdited = false;
```

Comment:
actually I think we should save whenever we have an input (throttled of course); but not now.

Also ideally I'd like to see if we can just delete this.draft usage -- but not sure if it's possible. Maybe we'd still need the draft id?